vm.c

```diff
@@ -33,6 +33,7 @@
 #include "internal/variable.h"
 #include "iseq.h"
 #include "rjit.h"
+#include "symbol.h"
```

Comment:
```diff
diff --git a/common.mk b/common.mk
index b5b1de7..[8](https://github.com/ruby/ruby/actions/runs/12188248512/job/34000852273?pr=12272#step:11:9)dd1f16 100644
--- a/common.mk
+++ b/common.mk
@@ -2004[9](https://github.com/ruby/ruby/actions/runs/12188248512/job/34000852273?pr=12272#step:11:10),6 +20049,7 @@ vm.$(OBJEXT): {$(VPATH)}rubyparser.h
 vm.$(OBJEXT): {$(VPATH)}shape.h
 vm.$(OBJEXT): {$(VPATH)}st.h
 vm.$(OBJEXT): {$(VPATH)}subst.h
+vm.$(OBJEXT): {$(VPATH)}symbol.h
 vm.$(OBJEXT): {$(VPATH)}thread_$(THREAD_MODEL).h
 vm.$(OBJEXT): {$(VPATH)}thread_native.h
 vm.$(OBJEXT): {$(VPATH)}variable.h
```

https://github.com/ruby/ruby/actions/runs/12188248512/job/34000852273?pr=12272

Reply:
Thank you!